### PR TITLE
Make some SSL issues out of scope for the bounty

### PR DESCRIPTION
**Pull request by nashe**
Let's try to make something clear and give some examples.

### CODE REVIEW

File: www/appendices/security-program.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ We take security seriously, and we're proud to be able to offer bounties through
 * [https://grtp.co](https://grtp.co) (not in scope for clickjacking)
 * any other [software we publish](https://github.com/gratipay)
 
+We target an "A" grade on SSLLabs for both [grtp.co](https://www.ssllabs.com/ssltest/analyze.html?d=grtp.co) and [gratipay.com](https://www.ssllabs.com/ssltest/analyze.html?d=gratipay.com).
```
Comment (nashe) [inline www/appendices/security-program.md]: 👍

**Comment by nashe** [comment]
@TheHmadQureshi, do you want to add something?

**Comment by whit537** [comment]
@nashe I don't think we should get into listing specific examples in the security program—that's what the "No Risk" category on [disclosures](http://inside.gratipay.com/appendices/disclosures) is for.

You good with a350afda748237d57a98e7775682c9ea6fd3f24a?

**Comment by whit537** [comment]
> You good with a350afd?

Seeing https://github.com/gratipay/inside.gratipay.com/pull/721#discussion_r70826573 now ... :-)

**Comment by nashe** [comment]
You're right. Completely OK with your edit :-)

**Comment by whit537** [comment]
Ready to merge once Travis is green ...

**Pull request merged**